src/activy/connectStrava.py

```
@@ -65,48 +64,58 @@ def connectStravaStage4(device, templates):
     """
     Stage 402: Decline synchronisation
     """
     clickNodeByClassInstance(device, "android.widget.Button", 0)
 
     try:
-        tryCheckStage(device, 402, templates)
-        connectStravaStage403(device, templates)
+        stage = tryCheckStage(device, ["4", "402or4o403"], templates)
+        if stage[0] == "402or4o403":
+            connectStravaStage403(device, templates)
     except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)